api/v1beta1/cinderbackup_types.go

```diff
@@ -132,5 +133,5 @@ func init() {
 
 // IsReady - returns true if service is ready to serve requests
 func (instance CinderBackup) IsReady() bool {
-	return instance.Status.ReadyCount >= 1
+	return instance.Status.ReadyCount >= 1 || instance.Spec.Replicas == 0
```

Comment:
This doesn't sound right to me.  Not only would the name be misleading with this change, but also the comment with the description of the method would be wrong, as we could return `true` when there is no service to serve requests.

Reply:
You're right about this one, I wasn't sure since it could result in just hacking the IsReady() function. So here my question: do you think that we should update the controller (e.g. see [1] for cinder-volume) to not update the `Status` or do you think this PR doesn't make sense?

[1] https://github.com/openstack-k8s-operators/cinder-operator/blob/master/controllers/cindervolume_controller.go#L120